integration_test/sql/multiple_pools.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Ecto.Integration.MultiplePoolsTest do
```

Comment:
Let's move this to `integration_test/cases/pool.exs` because I would like to enforce this on all adapters!